api_specifications/application-api-0.1.0.yaml

```diff
@@ -245,7 +253,7 @@ components:
         createdDate:
           type: string
           format: datetime
-          description: the date on which the order was created. Can be treated as the start date also
+          description: the date on which the application was created
```

Comment:
Add that this will be epoch time in the description. Minor capitalization fixes.